flink-runtime/src/main/java/org/apache/flink/runtime/webmonitor/stats/Statistics.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.webmonitor.stats;
+
+/** Represents one or more statistics samples. */
+public interface Statistics {
```

Comment:
I'm not sure if we really need this interface, and the generic parameter for the ThreadInfoOperatorTracker for that matter.

Reply:
The idea was mostly to surface the common pattern of sampling individual Operator subtasks and assembling the results (`OperatorStatsTracker`). I think if we continue to develop our debugging toolkit, such need will, most probably, arise again in the future.

Reply:
I left as is.